apps/app/package.json

```
@@ -25,6 +25,7 @@
     "@config-plugins/detox": "^1.1.0",
     "@config-plugins/ffmpeg-kit-react-native": "^0.0.1",
     "@config-plugins/ios-stickers": "^0.0.4",
+    "@config-plugins/react-native-adjust": "file:../../packages/react-native-adjust",
```

```suggestion
    "@config-plugins/react-native-adjust": "0.0.1",
```

This should work as expected since the project is setup as a yarn workspace (published or not). Most of the magic takes place in the root `/package.json` if you're curious.

---

Indeed, my bad!